### ISSUE TITLE
Openscad fails to run properly because of empty strings appended to scadcmd

### ISSUE THREAD

**Issue by Cantareus** [revarbat/openscad_runner#2]
When I attempt to call run() it fails and stdout returns OpenSCAD help and nothing more.

I don't understand what the empty strings after --preview, -- render and --wireframe are for but when I remove them everything works.

```
            if self.render_mode == RenderMode.render:
                scadcmd.extend(["--render"])
            elif self.render_mode == RenderMode.preview:
                scadcmd.extend(["--preview"])
            elif self.render_mode == RenderMode.thrown_together:
                scadcmd.extend(["--preview", "throwntogether"])
            elif self.render_mode == RenderMode.wireframe:
                scadcmd.extend(["--render"])
```

**Comment by revarbat** [comment]
This is apparently a Windows issue.  In Linux and MacOS, the blank arguments appear to be necessary.

**Comment by revarbat** [comment]
Implemented in PR #1
